### PR TITLE
Add ability to specify time as delta in seconds for Cylera

### PR DESCRIPTION
**Pull request by bodubas**
1. Added ability to specify relative time in seconds for time parameters, e.g. cylera_last_seen_after=60s
2. Reversed VCR back as tests are failing without it.

### CODE REVIEW

File: tasks/connectors/cylera/lib/client.rb
Comment (jgarber-cisco) [inline tasks/connectors/cylera/lib/client.rb:1]: Why is VCR required in the client?

File: spec/tasks/connectors/cylera/cylera_task_spec.rb [hunk 1]
```diff
@@ -50,5 +50,23 @@
         expect { task.run(options) }.to raise_error(SystemExit) { |e| expect(e.status).to_not be_zero }
       end
     end
+
+    context 'when time value is unexpected' do
```
Comment (jgarber-cisco) [inline spec/tasks/connectors/cylera/cylera_task_spec.rb:54]: "when time value is an unexpected format"?

File: tasks/connectors/cylera/lib/vcr.rb
Comment (jgarber-cisco) [inline tasks/connectors/cylera/lib/vcr.rb:1]: I think I deleted this file once before and everything was fine. 😁  VCR should only be used in specs, right?

**Review by jgarber-cisco — CHANGES_REQUESTED** [review]

File: spec/tasks/connectors/cylera/cylera_task_spec.rb (hunk 1 above)
Comment (bodubas) [reply spec/tasks/connectors/cylera/cylera_task_spec.rb]: Updated

**Review by jgarber-cisco — APPROVED** [review]
Can we remove the VCR thing next week? I really don't like it being in lib. 
When I removed it before, I thought the tests passed because it was in `spec/rspec_helper.rb`. Let's work on it together if not.

**Comment by bodubas** [comment]
> Can we remove the VCR thing next week? I really don't like it being in lib. When I removed it before, I thought the tests passed because it was in `spec/rspec_helper.rb`. Let's work on it together if not.

Sure, thanks for the review. Will work on this next week.

**Pull request merged**